src/Components/Blazor/testassets/Wasm.Authentication.Server/Wasm.Authentication.Server.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <UseLatestAspNetCoreReference>true</UseLatestAspNetCoreReference>
+    <DisableImplicitComponentsAnalyzers>true</DisableImplicitComponentsAnalyzers>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" />
+    <Reference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" />
+    <Reference Include="Microsoft.AspNetCore.Identity.UI" />
+    <Reference Include="Microsoft.EntityFrameworkCore.Relational" />
+    <Reference Include="Microsoft.EntityFrameworkCore.SQLite" />
+    <Reference Include="Microsoft.EntityFrameworkCore.Tools" />
+    <Reference Include="Microsoft.AspNetCore.Blazor.Server" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Wasm.Authentication.Client\Wasm.Authentication.Client.csproj" />
+    <ProjectReference Include="..\Wasm.Authentication.Shared\Wasm.Authentication.Shared.csproj" />
+  </ItemGroup>
+
+  <Target Name="GetStaticWebAssetsManifest" Returns="@(_SWAManifest)" AfterTargets="GenerateStaticWebAssetsManifest">
+    <ItemGroup>
+      <_SWAManifest Include="$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\$(_GeneratedStaticWebAssetsDevelopmentManifest)'))" />
```

Comment:
Could you add a comment explaining why this is here? Presumably it won't also appear in the real project template.

Reply:
Got rid of this